utils.py

```diff
@@ -0,0 +1,39 @@
+'''A set of utility functions
+'''
+import json
+
+
+def load_data(data_path):
+    '''
+    Load JSON data from a file.
+
+    Parameters
+    ----------
+    data_path : str
+        The path to the JSON file.
+
+    Returns
+    -------
+    dict
+        A Python dictionary representing the JSON data.
+    '''
+    with open(data_path, 'r', encoding='utf-8') as file:
```

Comment:
We should also do something to show if something goes wrong here, i.e. returning an error that can be relayed back to the user as 500 internal error